string blocker_id    # 造成阻塞的无人机ID
string blocked_id   # 被阻塞的无人机ID
bool is_bilateral   # true表示双向死锁，false表示单向阻塞